config.yaml

```diff
@@ -4,6 +4,10 @@
 # See https://juju.is/docs/config for guidance.
 
 options:
+  addons:
```

Comment:
How would we handle the case of arguments on addons?

Reply:
Added documentation for all existing config options.